src/components/HomePage/Banner.js

```diff
@@ -68,8 +68,10 @@ const useStyles = makeStyles(theme => ({
 		justifyContent: 'center',
 		alignItems: 'baseline',
 		alignSelf: 'flex-end',
+		padding: '25px 0px 0px 0px',
 		[theme.breakpoints.down('sm')]: {
-			alignSelf: 'flex-start'
+			alignSelf: 'flex-start',
+			padding: '15px'
```

Comment:
Similar fix for padding. Use `theme.spacing()` and set to spacing to the closest factor of 8 (which is 16px). If you need a refresher on how to convert `16px` padding to `theme.spacing()`, check out the [spacing MUI Docs](https://mui.com/customization/spacing/).